tests/test_model.py

```diff
@@ -0,0 +1,82 @@
+import os
+import uuid
+
+import pytest
+import pytest_asyncio
+
+from langchain_google_alloydb_pg.engine import AlloyDBEngine
+from langchain_google_alloydb_pg.model import AlloyDBModel
+
+
+def get_env_var(key: str, desc: str) -> str:
+    v = os.environ.get(key)
+    if v is None:
+        raise ValueError(f"Must set env var {key} to: {desc}")
+    return v
+
+
+EMBEDDING_MODEL_NAME = "textembedding-gecko@003" + str(uuid.uuid4()).replace("-", "_")
+
+
+@pytest.mark.asyncio
+class TestAlloyDBModel:
+    @pytest.fixture(scope="module")
+    def db_project(self) -> str:
+        return get_env_var("PROJECT_ID", "project id for google cloud")
+
+    @pytest.fixture(scope="module")
+    def db_region(self) -> str:
+        return get_env_var("REGION", "region for AlloyDB instance")
+
+    @pytest.fixture(scope="module")
+    def db_cluster(self) -> str:
+        return get_env_var("CLUSTER_ID", "cluster for AlloyDB")
+
+    @pytest.fixture(scope="module")
+    def db_instance(self) -> str:
+        return get_env_var("INSTANCE_ID", "instance for AlloyDB")
+
+    @pytest.fixture(scope="module")
+    def db_name(self) -> str:
+        return get_env_var("DATABASE_ID", "database name on AlloyDB instance")
+
+    @pytest_asyncio.fixture(scope="module")
+    async def engine(self, db_project, db_region, db_cluster, db_instance, db_name):
+        engine = await AlloyDBEngine.afrom_instance(
+            project_id=db_project,
+            cluster=db_cluster,
+            instance=db_instance,
+            region=db_region,
+            database=db_name,
+        )
+        print("START")
+        yield engine
+        print("END")
```

Comment:
```suggestion
        yield engine
```

Reply:
This file is supposed to be deleted and not a part of the PR anymore.